### ISSUE TITLE
add tags to ____ function

### ISSUE THREAD

**Issue by JMurphyWeb** [postgres-plugin/tags-system#45]
Every challenge, organisation that is created needs to be able to have tags added to it.

Possible interface:
```js
pg.tags.addTags(tableName <str>, itemId <int>, tagIds <arr>, function (..){..});
```

**Comment by JMurphyWeb** [comment]
```js
// addTags('challenges', 0, [0, 3, 9], function (err, res) { ... })
// adds following pairs to the tags_challenges table:
// challenges_id | tag_id
// 0             | 0
// 0             | 3
// 0             | 9
```

**Comment by JMurphyWeb** [comment]
I want to work out how to concatenate sql 'insert' queries

**Comment by SimonLab** [comment]
resolved with #50
